test/packages/same_version_twice/changelog.yml

```diff
@@ -0,0 +1,21 @@
+# newer versions go on top
+- version: "0.0.3"
+  changes:
+    - description: Third
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/193
+- version: "0.0.2"
+  changes:
+    - description: Second, but it's an error
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/193
+- version: "0.0.2"
+  changes:
+    - description: Second
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/193
+- version: "0.0.1"
+  changes:
+    - description: First
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/193
```

Comment:
Is there any test case with a version block with multiple changes?

Reply:
Good catch! I added multiple entries in few changelogs.

Reply:
Thanks!